src/actions/AddSiteActions.js

```diff
@@ -120,19 +116,24 @@ export const attemptToUploadSite = ({title, description, directions, nearestTown
                 nearestTown,
                 accessibility,
                 facilities,
+                features,
                 price,
                 coordinate
             })
             .then(() => {
                 dispatch({
                     type: ADD_SITE_SUCCESS
                 });
+
+                navigate(navKeys.ADD_SITE);
             })
             .catch(error => {
                 dispatch({
                     type: ADD_SITE_FAILURE,
                     payload: {error}
                 });
+
+                navigate(navKeys.ADD_SITE);
```

Comment:
Whether we succeed or fail, a user should be routed back to `ADD_SITE` after submitting  a site.